GameData/000_FilterExtensions Configs/SubCategories_LBP.cfg

```diff
@@ -0,0 +1,56 @@
+SUBCATEGORY
+{
+	name = Modern Ships
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = title
+			value = modern,dg,medium,tri,tumble,vette
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = Carrier Parts
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = title
+			value = carrier
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = WWII Hulls
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = folder
+			value = LShipPartsWW2
+		}
+	}
+}
+
+SUBCATEGORY
+{
+	name = Subs
+	icon = boat
+	FILTER
+	{
+		CHECK
+		{
+			type = folder
+			value = sub*
```

Comment:
You want path, folder is only for the GameData root folder, path just matches from the GameData root
```
type = path
value = LShipPartsModern/sub
```